### PR TITLE
[I18N][13.0]sale_coupon: translation

### PR DESCRIPTION
**Pull request by thanhluan240287**
Description of the issue/feature this PR addresses:

issue: https://github.com/Viindoo/customer-htv-distributor/issues/74#event-6115016711
Dịch module sale_coupon

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_coupon/i18n/vi.po [hunk 1]
```diff
@@ -30,13 +30,13 @@ msgstr ""
 #: code:addons/sale_coupon/models/sale_coupon_reward.py:0
 #, python-format
 msgid "%s %s discount on total amount"
-msgstr ""
+msgstr "%s %s chiết khấu theo tổng tiền"
```
Comment (royle-vietnam) [inline addons/sale_coupon/i18n/vi.po]:
```suggestion
msgstr "%s %s chiết khấu trên tổng số tiền"
```

File: addons/sale_coupon/i18n/vi.po [hunk 2]
```diff
@@ -30,13 +30,13 @@ msgstr ""
...
 
 #. module: sale_coupon
 #: code:addons/sale_coupon/models/sale_coupon_reward.py:0
 #, python-format
 msgid "%s%% discount on %s"
-msgstr ""
+msgstr "%s%% chiết khấu theo %s"
```
Comment (royle-vietnam) [inline addons/sale_coupon/i18n/vi.po]:
```suggestion
msgstr "%s%% chiết khấu trên %s"
```

File: addons/sale_coupon/i18n/vi.po [hunk 3]
```diff
@@ -70,13 +70,13 @@ msgstr "10% Chiết khấu"
...
 msgid "%s, a coupon has been generated for you"
-msgstr ""
+msgstr "%s, một phiếu giảm giá đã được tạo cho bạn"
 
 #. module: sale_coupon
 #: model:product.product,name:sale_coupon.product_product_10_percent_discount
 #: model:product.template,name:sale_coupon.product_product_10_percent_discount_product_template
 msgid "10.0% discount on total amount"
-msgstr ""
+msgstr "10.0% chiết khấu theo tổng tiền"
```
Comment (royle-vietnam) [inline addons/sale_coupon/i18n/vi.po]:
```suggestion
msgstr "10.0% chiết khấu trên tổng số tiền"
```

File: addons/sale_coupon/i18n/vi.po [hunk 4]
```diff
@@ -228,12 +319,12 @@ msgstr "Một phiếu giảm giá đã được áp dụng cho cùng một phầ
 #: code:addons/sale_coupon/models/sale_coupon_program.py:0
 #, python-format
 msgid "A minimum of %s %s should be purchased to get the reward"
-msgstr ""
+msgstr "Tối thiểu %s %s cần mua để nhận thưởng"
```
Comment (royle-vietnam) [inline addons/sale_coupon/i18n/vi.po]:
```suggestion
msgstr "Giá trị tối thiểu %s %s nên mua để nhận phần thưởng"
```

File: addons/sale_coupon/i18n/vi.po [hunk 5]
```diff
@@ -228,12 +319,12 @@ msgstr "Một phiếu giảm giá đã được áp dụng cho cùng một phầ
...
 
 #. module: sale_coupon
 #: model:ir.model.fields,help:sale_coupon.field_sale_coupon_program__active
 msgid "A program is available for the customers when active"
-msgstr ""
+msgstr "Một chương trình có sẵn cho khách hàng khi hoạt động"
```
Comment (royle-vietnam) [inline addons/sale_coupon/i18n/vi.po]:
```suggestion
msgstr "Một chương trình khả dụng cho khách hàng khi kích hoạt"
```

File: addons/sale_coupon/i18n/vi.po [hunk 6]
```diff
@@ -228,20 +319,23 @@ msgstr "Một phiếu giảm giá đã được áp dụng cho cùng một phầ
 #: code:addons/sale_coupon/models/sale_coupon_program.py:0
 #, python-format
 msgid "A minimum of %s %s should be purchased to get the reward"
-msgstr ""
+msgstr "Giá trị tối thiểu %s %s nên mua để nhận phần thưởng"
```
Comment (vietduc1989) [inline addons/sale_coupon/i18n/vi.po:322]:
```suggestion
msgstr "Giá trị tối thiểu %s %s cần mua để nhận phần thưởng"
```

File: addons/sale_coupon/i18n/vi.po [hunk 7]
```diff
@@ -1067,7 +1186,7 @@ msgstr "Chương trình khuyến mãi"
 #. module: sale_coupon
 #: model:ir.model.fields.selection,name:sale_coupon.selection__sale_coupon_program__program_type__promotion_program
 msgid "Promotional Program"
-msgstr ""
+msgstr "Chương trình khuyến mại"
```
Comment (vietduc1989) [inline addons/sale_coupon/i18n/vi.po]:
```suggestion
msgstr "Chương trình khuyến mại"
```
Cần thống nhất các chỗ là khuyến mại hay khuyến mãi

**Pull request merged**